hail/Makefile

```
@@ -129,9 +129,9 @@ check-pypi:
 HAIL_TWINE_CREDS_FOLDER ?= /secrets/
 
 .PHONY: pypi-deploy
-pypi-deploy: export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username)
-pypi-deploy: export TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password)
 pypi-deploy: check-pypi wheel
+	TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username) \
+	TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password) \
```

I don't think this syntax is right. If we're going to use the inside of a rule to run this, then I think that the syntax needs to be shell, which needs to not have spaces or the colon for variable assignment.
```suggestion
	TWINE_PASSWORD=$(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password) \
```

---

oh, cool, neat "suggestion" feature. And yes you're totally right.

---

I did have it right the first time: https://github.com/hail-is/hail/pull/6318#issuecomment-501313676

Clearly I am a very thorough reviewer 🙍‍♂